### PR TITLE
bayestestR_distribution

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
- [x] fix link to `distribution` docs
- [x] smoothen the tone

### CODE REVIEW

File: content/posts/bayestestR_distribution.Rmd [hunk 1]
```diff
@@ -52,24 +52,24 @@ x %>%
...
+Thus, please note that the `rnorm_perfect` name will be deprecated in the next version, and ultimately removed to avoid further confusion :)
 
 
 ## Any other suggestions?
 
-Don't forget, the [easystats project](https://github.com/easystats) (that includes bayestestR) is very open to contributions and suggestions!
+Don't forget, the [easystats project](https://github.com/easystats) (that includes `bayestestR`) is very open to contributions and suggestions!
 
-Thus, if you ever plan on publishing personal correspondence with the response to a falsely naive question, please remember that you can always [make suggestions](https://github.com/easystats/bayestestR/issues) and contribute to the package, to *actually help* improving it, with the spirit of collaboration, and in the tradition of open science ;)
+Thus, if you ever plan on publishing personal correspondence of a response to a falsely naive question, please remember that you can always [make suggestions](https://github.com/easystats/bayestestR/issues) and contribute to the package, to *actually help* improving it, with the spirit of collaboration, and in the tradition of open science ;)
```
Comment (pdwaggoner) [inline content/posts/bayestestR_distribution.Rmd]: I’m not convinced this last section is needed. I get the idea but it sounds a little vindictive. I think the previous line effectively opens the opportunity for contributions. Just my opinion.

File: content/posts/bayestestR_distribution.Rmd [hunk 2]
```diff
@@ -35,9 +35,9 @@ x %>%
...
+This function's name was initially chosen because of its proximity (argument-wise) with its random counterpart, `rnorm` (*i.e.*, it has the same arguments, the same order and the same output type). However, we [quickly realised](https://github.com/easystats/bayestestR/issues/86) this this name was not perfect (*no pun intended*), as indeed the obtained distribution was not *random* (and the `r` in `rnorm` stands for *random*).
 
-Thus, after discussion, we decided to change it to `distribution_normal` (INSERT LINK). And guess what, we also added other types of "perfect" distributions:
+Thus, after discussion, we decided to change it to `distribution_normal` (INSERT LINK). And guess what, we also used that opportunity to add other types of "perfect" distributions:
```
Comment (pdwaggoner) [inline content/posts/bayestestR_distribution.Rmd]: I’d take out “guess what”

**Comment by pdwaggoner** [comment]
Looks mostly good to me. I just made a couple suggestions. Happy to keep reviewing a new version if needed. It’s always tough getting this kind of criticism. But I think it’s a great opportunity to clarify functions for interested users.

File: content/posts/bayestestR_distribution.Rmd (hunk 1 above)
Comment (DominiqueMakowski) [reply content/posts/bayestestR_distribution.Rmd]: Yes this has been removed 😅

**Review by strengejacke — COMMENTED** [review]
Posts looks good! I just would remove the emphasize on "actually help" at the end of the post.

**Comment by DominiqueMakowski** [comment]
If everybody agrees, I'll release this on Monday :)

**Comment by pdwaggoner** [comment]
Looks great to me - thanks for making the small changes. The tone overall is great and the post is very helpful at clarifying (via responding to that guy).

**Pull request merged**